clientdb/order.go

```diff
@@ -65,6 +66,13 @@ var (
 	orderTlvKey = []byte("order-tlv")
 )
 
+// IsNotFoundErr tries to match the given error with ErrNoOrder.
+func IsOrderNotFoundErr(err error) bool {
```

Comment:
It looks like we are not using grpc error codes. I am not sure if there is any other way to check for what type of error we are receiving from the server that string matching here...

Reply:
Yeah, we currently don't use any custom codes on the server side. String matching is okay here. Though I'm not sure if this function better fits into the `auctioneer` package?